pkg/log/auditlog/auditlog.go

```diff
@@ -50,6 +51,7 @@ func InitLogger(owningServiceName string, coverageLevel string, logWriter io.Wri
 	// Initialize the Logger with the given coverage level
 	l := logger{
 		owningServiceName: owningServiceName,
+		enabled:           true,
```

Comment:
```suggestion
		enabled:           false,
```

set the default to false